README.md

```diff
@@ -17,10 +17,10 @@ cd <catkin workspace location>/src
 git clone https://github.com/uwrobotics/uwrt_mars_rover.git
 
 # Install upstream source dependencies
-rosinstall --catkin . uwrt_mars_rover/upstream_dependencies.rosinstall
+sudo rosinstall --catkin . uwrt_mars_rover/upstream_dependencies.rosinstall
```

Comment:
are these not the same changes in fix rosinstall #53 ?

Reply:
this branched off that. rebasing this one so the diff makes sense now